### PR TITLE
Create KycDocumentRefusedReasonType class

### LINKED ISSUE #229: Missing class KycDocumentRefusedReasonType
From sandbox dashboard :

- Document unreadable
- Document not accepted
- Document has expired
- Document incomplete
- Document missing
- Document does not match user data
- Document does not match account data
- Specific case
- Document falsified
- Underage person
- Trigger PEPS
- Trigger sanaction lists
- Trigger interpol

### PR DESCRIPTION
**Pull request by gaetan-petit**
Fix #229

### CODE REVIEW

**Comment by gaetan-petit** [comment]
Not really sure this don't pass the test, though.

File: MangoPay/KycDocumentRefusedReasonType.php [hunk 1]
```diff
@@ -0,0 +1,22 @@
+<?php
+namespace MangoPay;
+
+/**
+ * Kyc Refused Reason Type
+ */
+class KycDocumentRefusedReasonType
+{
+    const DocumentUnreadable = 'DOCUMENT_UNREADABLE';
```
Comment (iammichiel) [inline MangoPay/KycDocumentRefusedReasonType.php]: In my opinion, const should be expressed in uppercase with underscores. This is however a purely checkstyle opinion. I'll leave the core-devs decide.

Comment (gaetan-petit) [reply MangoPay/KycDocumentRefusedReasonType.php]: Agree, but I followed the coding standard featured in this library.

Comment (mickaelpois) [reply MangoPay/KycDocumentRefusedReasonType.php]: @iammichiel @gaetan-petit Good point, this definitely should be. I'll keep that in mind for some next major update !

**Pull request merged**